src/revpi-factory-reset

```diff
@@ -62,13 +71,18 @@ fi
 
 if [ "$kernel" = 49 ] ; then
```

Comment:
Just remove the 44/49 detection all together. It doesn't serve any purpose any more.